### PR TITLE
PLANET4-5222 P4 settings grouped and divided to subpages

### PR DESCRIPTION
**Pull request by koyan**
Group the Planet4 settings into groups.
Note: The page that had nothing to do with the settings (dev report), has not been touched. The links here are just links towards the existing pages

Edit: Removed the media library and planet4-dev-report links (There is MR on the media library plugin directly but it should be merged AFTER this one)
Note2: The Media library MR has to be merged AFTER this has gone to production

### CODE REVIEW

**Comment by koyan** [comment]
Related to https://github.com/greenpeace/planet4/issues/68

**Comment by koyan** [comment]
Related MR in the Media Library: https://github.com/greenpeace/planet4-plugin-medialibrary/pull/46

File: src/Settings.php [hunk 1]
```diff
@@ -346,10 +392,21 @@ public function init() {
...
+	public function add_options_pages() {
+		$this->options_page = add_menu_page( $this->title, $this->title, 'manage_options', $this->key, function () {}, 'dashicons-admin-site-alt' );
+		foreach ( $this->subpages as $path => $subpage ) {
+			add_submenu_page(
+				$this->key,
+				$subpage['title'],
+				$subpage['title'],
+				'manage_options',
+				$path,
+				[ $this, 'admin_page_display' ]
```
Comment (lithrel) [inline src/Settings.php:404]: You can avoid using `global` in `admin_page_display` by changing this callback to 
```php
				$path,
				function () use ( $path ) {
					self::admin_page_display( $path );
				}
			);
```
and changing `admin_page_display()` signature to 
```php
	public function admin_page_display( string $plugin_page ) {
```

**Comment by comzeradd** [comment]
Minor semantic change: can you also change the menu entry to "Planet 4" (with a space)?

**Pull request merged**